### ISSUE TITLE
add subpage to let users create csv line themselves

### ISSUE THREAD

**Issue by trutzig89182** [trutzig89182/Mastodon-Sociologists#23]
This can be of some help in the short run to save some time when adding new people. They just create their csv line so its only one copy & past step instead of multiple.

Still some things to fix.

- [ ] url input type not enforced, why?
- [ ] santize text, exlcude htlm tags on input
- [x] replace child instead of append 
- [ ] copy to clipboard (button or automatic?)
- [x] add radio button: "I already am included and want to update my info"
- [ ] make mailto work for smartphone

in Branch: https://github.com/trutzig89182/Mastodon-Sociologists/tree/wip-004_input-form

add-user-string.html
resources/js/add-user-string.js

Put the js seperatly for now, but already adapted to app.js form. 

@nathanlesage fyi

**Comment by nathanlesage** [comment]
Puhh, that looks a little bit clumsy. I see what you want to do, but I have to check if I can maybe come up with a better idea.

**Comment by trutzig89182** [comment]
Everything I do is clumsy. But some of it is quite helpful to me. ;)

I made this work the way I want it to, but it is still a mess. I will let it run on my server to still already profit from it, but will still take some time to clean it up.

I also stumbled over another problem. If we use e-mail more: how do we assure that people are handing in their own accounts? I included a preliminary classical method here: Make them do somthing with the account. They have to put #SoM in a post and include the link to that post in the form. This could possibly also made more elegant, but at lest it works somehow. 

(also already receives some e-mails, and that’s really better than via Mastodon.)
